omod-2.0/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource2_0.java

```diff
@@ -1,17 +1,205 @@
 package org.openmrs.module.attachments.rest;
 
+
+import java.util.Arrays;
+import java.util.List;
+import org.hibernate.FlushMode;
```

Comment:
thanks @sacull ,Actually am using this import, let me know if there is something else please

Reply:
Oddly, I'm sure these two imports weren't used before and the `save(Attachement)` method looked different, however, I can't find it in history. Maybe something just happened to me. Bravo for vigilance. ;)

Reply:
Thank you too @sacull , if you approve this , then we can ping @dkayiwa  for final remarks ,actually after this merge, am releasing attachment module asap

Reply:
After the last commit (464f5aa) the `save(Attachment)` method looks like I remembered it, so again these imports are not used:
![image](https://user-images.githubusercontent.com/9057882/94605791-e29b1c00-0299-11eb-913f-924e752cd259.png)

Reply:
Are you suggesting we get rid of that method of save attachments

Reply:
In no case. I'm just trying to tell you that by removing this code marked in red, you make some imports stop being used because the code that uses them disappears. :P
I mean here:
- org.hibernate.FlushMode;
- org.openmrs.module.emrapi.db.DbSessionUtil.

Reply:
True @sacull, shall we please leave them there  at the moment, then we shall create a ticket that will cater for that refactoring as you have been doing thanks

Reply:
Hey @sacull, shall we go with this idea??

Reply:
@sherrif10 It's not my decision whether to leave it or not, I'm not important enough here. ;)

In addition, I would like to take a better look at this PR to be able to click "Approve" with full responsibility, unfortunately, until the end of the week, I am not able to do it due to lack of time. I'm sorry.

Reply:
haa you are more important @sacull, we have learnt clean code from you , keep up

Reply:
Did you do as @sacull suggested above?

Reply:
Yes i responded to his comments above